dotnet/samples/Demos/BedrockTest/Program.cs

```diff
@@ -0,0 +1,141 @@
+// // var kernel3 = Kernel.CreateBuilder().AddBedrockTextGenerationService("cohere.command-text-v14").Build();
+// // var kernel3 = Kernel.CreateBuilder().AddBedrockTextGenerationService("cohere.command-r-plus-v1:0").Build();
+// var kernel3 = Kernel.CreateBuilder().AddBedrockTextGenerationService("meta.llama3-8b-instruct-v1:0").Build();
+//
+// // AI21Labs Jurassic and Jamba models do not support streaming
+// // var kernel3 = Kernel.CreateBuilder().AddBedrockTextGenerationService("ai21.j2-ultra-v1").Build();
+// // var kernel3 = Kernel.CreateBuilder().AddBedrockTextGenerationService("ai21.jamba-instruct-v1:0").Build();
+//
+// var streamTextGenerationService = kernel3.GetRequiredService<ITextGenerationService>();
+// var streamTextGeneration = streamTextGenerationService.GetStreamingTextContentsAsync(UserPrompt3).ConfigureAwait(true);
+// await foreach (var textContent in streamTextGeneration)
+// {
+//     Console.Write(textContent.Text);
+// }
+
+
+
+
+
+
+// ----------------------------STREAM CHAT COMPLETION----------------------------
+// string userInput;
+// ChatHistory chatHistory = new ChatHistory();
+// do
+// {
+//     Console.Write("Enter a prompt (or 'exit' to quit): ");
+//     userInput = Console.ReadLine();
+//
+//     if (userInput.ToLower() != "exit")
+//     {
+//         chatHistory.AddMessage(AuthorRole.User, userInput);
+//         // Console.WriteLine($"Stream Chat Completion Question: {userInput}");
+//
+//         // var kernel4 = Kernel.CreateBuilder().AddBedrockChatCompletionService("mistral.mistral-7b-instruct-v0:2").Build();
+//         // var kernel4 = Kernel.CreateBuilder().AddBedrockChatCompletionService("amazon.titan-text-premier-v1:0").Build();
+//         // var kernel4 = Kernel.CreateBuilder().AddBedrockChatCompletionService("anthropic.claude-v2").Build();
+//         // var kernel4 = Kernel.CreateBuilder().AddBedrockChatCompletionService("anthropic.claude-3-sonnet-20240229-v1:0").Build();
+//         // var kernel4 = Kernel.CreateBuilder().AddBedrockChatCompletionService("cohere.command-r-plus-v1:0").Build(); //FAILING
+//         var kernel4 = Kernel.CreateBuilder().AddBedrockChatCompletionService("meta.llama3-8b-instruct-v1:0").Build();
+//
+//         // AI21 Labs does not support streaming
+//         // var kernel4 = Kernel.CreateBuilder().AddBedrockChatCompletionService("ai21.jamba-instruct-v1:0").Build();
+//
+//
+//
+//         var chatCompletionService = kernel4.GetRequiredService<IChatCompletionService>();
+//         var result = chatCompletionService.GetStreamingChatMessageContentsAsync(chatHistory).ConfigureAwait(false);
+//         string output = "";
+//         await foreach (var message in result)
+//         {
+//             Console.Write($"{message.Content}");
+//             output += message.Content;
+//         }
+//         Console.WriteLine();
+//         chatHistory.AddMessage(AuthorRole.Assistant, output);
+//     }
+// } while (userInput.ToLower() != "exit");
```

Comment:
It's not great to have a bunch of commented code, since it won't be able to be scanned or checked for build errors at compile time. It might be better to have the user input first a selection like 1 to 4 for which demo they want to do and then branch to that code.